t/08-autotest.t

```diff
@@ -341,6 +342,10 @@ subtest 'load test successfully when CASEDIR is a relative path' => sub {
 pass autotest::loadtest('tests/test.py'), 'can load python test module at all';
 loadtest 'test.py', 'we can also parse python test modules';
 
+stderr_like {
+    throws_ok { autotest::loadtest 'tests/faulty.py' } qr/py_eval raised an exception/, 'dies on Python exception';
+} qr/Traceback.*No module named.*thismoduleshouldnotexist.*/s, 'Python traceback logged';
```

Comment:
Running all tests from your branch for me this yields

```
Useless use of a constant ("can load python test module at a"...) in void context at ./08-autotest.t line 342.
[2021-09-23T19:18:39.301 CEST] [debug] scheduling test tests/test.py

#   Failed test 'dies on Python exception'
#   at ./08-autotest.t line 346.
# expecting: Regexp ((?^:py_eval raised an exception))
# found: error on tests/faulty.py: Can't open file "foo/tests/faulty.py": No such file or directory at (eval 256) line 5.
# BEGIN failed--compilation aborted at (eval 256) line 5.

#   Failed test 'Python traceback logged'
#   at ./08-autotest.t line 347.
# STDERR:
# loadtest needs a script below foo - tests/faulty.py is not
# [2021-09-23T19:18:39.311 CEST] [warn] !!! autotest::loadtest: error on tests/faulty.py: Can't open file "foo/tests/faulty.py": No such file or directory at (eval 256) line 5.
#   BEGIN failed--compilation aborted at (eval 256) line 5.
#   
# 
# doesn't match:
# (?^s:Traceback.*No module named.*thismoduleshouldnotexist.*)
# as expected
# Looks like you failed 2 tests of 93.
[19:18:39] ./08-autotest.t .......................... 
```

Reply:
Oh, looks like I forgot to check-in the Python file.